channels_auth_token_middlewares/middleware/drf.py

```diff
@@ -65,12 +87,55 @@ def __init__(self, *args, **kwargs):
         self._auth = JWTAuthentication()
         self._exceptions = (AuthenticationFailed, InvalidToken, TokenError)
 
-        return super().__init__(*args, **kwargs)
-
     @database_sync_to_async
-    def get_user_instance(self, token_key):
+    def get_jwt_user_instance(self, token_key):
+        if self._auth is None or self._exceptions is None:
+            raise RuntimeError("_setup method has to be called before.")
+
         try:
             validated_token = self._auth.get_validated_token(token_key)
             return self._auth.get_user(validated_token)
         except self._exceptions:
             return None
+
+
+class SimpleJWTAuthTokenMiddleware(
+    HeaderAuthTokenMiddleware,
+    SimpleJWTAuthTokenMiddlewareMixin,
+):
+    """
+    Simple JWT auth token middleware.
+
+    https://django-rest-framework-simplejwt.readthedocs.io/en/latest/index.html
+    """
+
+    header_name = "Authorization"
+    keyword = "Bearer"
+
+    def __init__(self, *args, **kwargs):
+        self._setup()
+        return super().__init__(*args, **kwargs)
+
+    async def get_user_instance(self, token_key):
+        return await self.get_jwt_user_instance(token_key)
+
+
+class QueryStringSimpleJWTAuthTokenMiddleware(
```

Comment:
Should the `docs/` be updated to include usage and examples for these new `Middleware`?

Reply:
Fixed in 6a929e4